gulpfile.js

```diff
@@ -79,8 +79,9 @@ gulp.task('tests:sauce:start', function (done) {
     tunnel.start(function (isCreated) {
         isTunnelCreated = isCreated;
         if (!isCreated) {
-            console.log('Failed to create Sauce tunnel, skipping tests');
-            done();
+            console.log('Failed to create Sauce tunnel, returning error code');
+            // force the process to exit with error code if couldn't create the tunnel
+            process.exit(1);
```

Comment:
@FinalAngel please see the live example - https://travis-ci.org/aldryn/aldryn-newsblog/jobs/81575034 - all tests are stopped